modelLINQ.Test/AnyTests.cs

```diff
@@ -0,0 +1,69 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Text;
+using System.Threading.Tasks;
+using static modelLINQ.Test.TestClasses;
+
+namespace modelLINQ.Test
+{
+    [TestClass]
+    public class AnyTests
+    {
+        private List<ObjectD> Objects { get; set; }
+
+        [TestInitialize]
+        public void Init()
+        {
+            Objects = new List<ObjectD>
+            {
+                new ObjectD
+                {
+                    Id = 1,
+                    ListOfA = new List<ObjectA>
+                    {
+                        new ObjectA
+                        {
+                            Id = 1
+                        }
+                    }
+                },
+                new ObjectD
+                {
+                    Id = 2
+                }
+            };
+        }
+
+        /// <summary>
+        /// Tests any method call against a list of objects
+        /// </summary>
+        [TestMethod]
+        public void TestAny()
```

Comment:
Optional but might suggest just adding AAA comments to make it clearer, e.g.:
```csharp
[TestMethod]
public void Withdraw_ValidAmount_ChangesBalance()
{
    // arrange
    double currentBalance = 10.0;
    double withdrawal = 1.0;
    double expected = 9.0;
    var account = new CheckingAccount("JohnDoe", currentBalance);

    // act
    account.Withdraw(withdrawal);

    // assert
    Assert.AreEqual(expected, account.Balance);
}
```

Reply:
It is a lot easier being lazy when no one reviews your PRs

Reply:
Added some comments